### PR TITLE
converted into urdu

### PR DESCRIPTION
**Pull request by mhkasif**
Page in question is at available in preview at https://deploy-preview-20--ur-reactjs.netlify.app/community/courses.html

### CODE REVIEW

**Comment by jabranr** [comment]
@mhkasif Thank you for translating this page.

Do you have a relevant issue for this already and if not then can you please create one? So that we can link it to the progress of the project. I will review the changes and will revert accordingly unless someone else does it before me.

**Comment by mhkasif** [comment]
Alright will check if there's amy existing relevant issue , otherwise will create one.
Thanks

File: content/community/courses.md [hunk 1]
```diff
@@ -1,51 +1,57 @@
...
-- [React Essentials for Designers](https://learnreact.design) - React courses tailored for designers: the fundamentals, capabilities, limitations and how they relate to design.
+- [React for Designers](https://designcode.io/react) - .کا کورس React  ڈیزائنرز کے ذریعہ ، ڈیزائنرز کے لئے 6 گھنٹے
 
-- [Essential React](https://learnreact.com/lessons/2018-essential-react-1-overview) - A crash course for doers, moving fast from "Hello World" to advanced component composition.
+- [ کے لیے Designers کی ضروریات  React](https://learnreact.design) - ڈیزائنرز کے لئے تیار کردہ کورسز: بنیادی اصول ، قابلیت ، حدود اور ان کا ڈیزائن سے کس طرح تعلق ہے React
 
-- [React Training: Advanced React.js](https://courses.reacttraining.com/p/advanced-react) - Take your React skills to the next level.
 
-- [Tyler McGinnis](https://tylermcginnis.com/courses) - Tyler McGinnis provides access to his courses for a monthly fee. Courses include "React Fundamentals" and "Universal React".
+- [React ضروری](https://learnreact.com/lessons/2018-essential-react-1-overview) - کوڈ لکھ کر سیکھنا پسند کرنے والوں کے لئے ایک تعلیمی نصاب ، یہ کورس انہیں جلدی سیکھنے اور جدید ترین درخواستیں بنانے کے قابل بنائے گا
```
Comment (mumairofficial) [inline content/community/courses.md:46]: In my opinion in this context correct word should be ایپلیکیشنز instead of درخواستیں

Comment (mhkasif) [reply content/community/courses.md]: yeah :)
if you get any other mistakes please point it out <3

**Comment by mhkasif** [comment]
@jabranr  Hey, can you look at the deployed page and tell me if everything working fine?

File: content/community/courses.md [hunk 2]
```diff
@@ -1,51 +1,57 @@
...
 layout: community
 sectionid: community
 permalink: community/courses.html
 ---
 
-## Free Courses {#free-courses}
+## مفت نصاب {#free-courses}
 
-- [Glitch: React Starter Kit](https://glitch.com/glimmer/post/react-starter-kit/) - A free, 5-part video course with interactive code examples that will help you learn React.
+- [Glitch: React Starter Kit](https://glitch.com/glimmer/post/react-starter-kit/) - انٹرایکٹو کود مثالوں کے ساتھ بالکل مفت، ۵ ھصے کا ویڈیو کورس جو آپ کو ری ایکٹ(React) سیکھنے میں مدد فراہم کرے گا ۔
```
Comment (jabranr) [inline content/community/courses.md:11]: - Interactive code = متعامل کوڈ
- No need to translate React into Urdu

I suggest a translation which is not more meaning to meaning but more readable i.e.

<div dir="rtl">
متعامل کوڈ کی مثالوں  کے ساتھ ۵ حصوں پر مبنی ایک مفت ویڈیو کورس جوآپ کو React سیکھنے میں مدد دے گا۔
</div>

File: content/community/courses.md [hunk 3]
```diff
@@ -1,51 +1,57 @@
...
 
-- [Codecademy: React 101](https://www.codecademy.com/learn/react-101) - Codecademy's introductory course for React.
+- [Codecademy: React 101](https://www.codecademy.com/learn/react-101) - کوڈ اکیڈمی کا ری ایکٹ (React) کے لیے تعارفی کورس ۔
```
Comment (jabranr) [inline content/community/courses.md:13]: Similarly you do not need to translate the names of other platforms i.e. Codeacademy

Comment (jabranr) [reply content/community/courses.md:13]: Also do not need to wrap React into parentheses

File: content/community/courses.md [hunk 4]
```diff
@@ -1,51 +1,57 @@
...
 
-- [Egghead.io: Start Learning React](https://egghead.io/courses/start-learning-react) - This series will explore the basic fundamentals of React to get you started.
+- [Egghead.io:ری ایکٹ(React) سیکھنا شروع کریں](https://egghead.io/courses/start-learning-react) - یہ سبق آموز سلسلہ آپ کو React  لائبریری کی بنیادی باتیں سکھائے گا ۔
```
Comment (jabranr) [inline content/community/courses.md:15]: Do not translate the headings for external resources i.e. `Egghead.io: Start Learning React` remains unchanged

Comment (jabranr) [reply content/community/courses.md:15]: This is true for all of headings at this page

File: content/community/courses.md (hunk 3 above)
Comment (mhkasif) [reply content/community/courses.md:13]: The problem is I'm having a syntax issue if I don't start with Urdu.
To overcome this I had to translate React and codeacademy in Urdu.

Comment (mhkasif) [reply content/community/courses.md:13]: Screenshot 2020-08-08 at 3 14 13 PM

here you can see the issue.

Comment (mhkasif) [reply content/community/courses.md:14]: @jabranr ??

Comment (jabranr) [reply content/community/courses.md:14]: That is something will be fixed when RTL is implemented correctly. For now you can wrap the line inside enforced RTL i.e.

`<span dir="rtl">YOUR TRANSLATION GOES HERE</span>`

Try this and see if that works for you.

**Review by jabranr — CHANGES_REQUESTED** [review]

**Review by mhkasif — COMMENTED** [review]
Bugs removed Accordingly

**Review by jabranr — APPROVED** [review]

**Comment by mhkasif** [comment]
not yet deployed

**Comment by jabranr** [comment]
> not yet deployed

Preview available at https://urreactjsorg-babe8cpqb-fbopensource.vercel.app/community/courses.html

File: content/community/courses.md [hunk 5]
```diff
@@ -1,55 +1,56 @@
...
-- [React Training: Advanced React.js](https://courses.reacttraining.com/p/advanced-react) - Take your React skills to the next level.
+- [React Training: Advanced React.js](https://courses.reacttraining.com/p/advanced-react)<span dir='rtl'> - React میں اپنی صلاحیتوں کو ترقی دیں ۔</span>
 
-- [Tyler McGinnis](https://tylermcginnis.com/courses) - Tyler McGinnis provides access to his courses for a monthly fee. Courses include "React Fundamentals" and "Universal React".
+- [Tyler McGinnis](https://tylermcginnis.com/courses)<span dir='rtl'> - Tyler McGinnis کے ساتھ سیکھیں ، جو ماہانہ تربیتی سیشن پیش کرتا ہے۔ ان کورسز میں بنیادی باتیں سیکھنا بھی  شامل ہے ۔</span>
 
-- [Mastering React](https://codewithmosh.com/p/mastering-react/) - Build professional interactive apps with React.
+- [Mastering React](https://codewithmosh.com/p/mastering-react/)<span dir='rtl'> - React لائبریری کے ساتھ انٹرایکٹو ایپلی کیشنز بنائیں ۔</span>
 
-- [React Tutorial](https://react-tutorial.app) - Learn React step by step in an interactive environment with flashcards.
```
Comment (armujahid) [inline content/community/courses.md:51]: Why this line is removed? We need to re-add this

Comment (jabranr) [reply content/community/courses.md:51]: Probably the outcome of a merge conflict resolution. Will revert it.

**Pull request merged**